test/user-agent/test/common.js

```diff
@@ -70,15 +71,26 @@ export const trustBlocStartupOpts = {
     agentUsername: `user-agent`
 }
 
-export async function loadAries(name) {
-    let opts = ariesStartupOpts
+export async function loadFrameworks(name, loadTrustblocOpts) {
+    let ariesOpts = ariesStartupOpts
     if (name) {
-        opts = JSON.parse(JSON.stringify(ariesStartupOpts))
-        opts["db-namespace"] = `${name}db`
-        opts["agent-default-label"] = `${name}-user-agent`
+        ariesOpts = JSON.parse(JSON.stringify(ariesStartupOpts))
+        ariesOpts["db-namespace"] = `${name}db`
+        ariesOpts["agent-default-label"] = `${name}-user-agent`
     }
 
-    return new Aries.Framework(opts)
+    if (loadTrustblocOpts) {
+        let trustblocOpts = trustBlocStartupOpts
+        if (name) {
+            trustblocOpts = JSON.parse(JSON.stringify(trustBlocStartupOpts))
```

Comment:
this logic doesn't make sense... why stringify and parse `trustBlocStartupOpts` again, and why `if (name)` ?

Reply:
I'm not sure why - that code was already there, I just renamed the function and added to it... @sudeshrshetty do you know?

Reply:
@DRK3 You followed same pattern as aries opts? you don't have to do this for trustbloc opts.
In aries opts, we parse & stringify to clone aries opts JS object, otherwise test specific customization of aries opts will conflict with other. 

You don't have to do this for trustbloc opts since I don't see anything being customized in trustbloc opts.

Reply:
@sudeshrshetty @llorllale Oops, yeah that shouldn't be there for the trustbloc opts... removed it.